src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.tsx

```diff
@@ -296,8 +305,12 @@ export const RespondentDataReview = () => {
           />
           {!isLoading && (
             <>
-              {selectedAnswer && responsesSummary && !error && (
-                <ResponsesSummary {...responsesSummary} data-testid={dataTestid} />
+              {selectedAnswer && responsesSummary && !!sourceSubject && !error && (
```

Comment:
Looks like a new test is failing in `RespondentDataReview.test.tsx`, maybe you're already on top of it. I'm guessing it's because the data for `sourceSubject` needs to get mocked into the response?

Reply:
@farmerpaul I removed that additional check because I assume the `sourceSubject` will always be returned from the answer object. I don't believe the whole summary should not return if that fails . Is that a correct assumption?

Reply:
Ah correct. Yes if for whatever reason the `sourceSubject` is missing we should still render the component. Does this fix that broken test?